modules/tinymce/src/core/test/ts/browser/SelectionOverridesTest.ts

```diff
@@ -37,10 +37,10 @@ UnitTest.asynctest('browser.tinymce.core.SelectionOverridesTest', function (succ
     const rect = editor.dom.getRect(firstTd);
 
     editor.fire('mousedown', {
-      target: firstTd,
+      target: firstTd as EventTarget,
```

Comment:
Is this needed given the other cast below?

Reply:
Yeah, without it the other cast doesn't work, as it views the `target` fields as incompatible types